iiif_prezi3/helpers/canvas_helpers.py

```diff
@@ -3,21 +3,21 @@
 
 
 class AddImageToCanvas:
-    """Adds an image to an existing canvas.
 
-    Args:
-        image_url (str): An HTTP URL which points to the image.
-        anno_id (str): An HTTP URL for the annotation to which the image will be
-            attached.
-        anno_page_id (str): An HTTP URL for the annotation page to which the
-            annotation will be attached.
+    def add_image(self, image_url, anno_id=None, anno_page_id=None):
+        """Adds an image to an existing canvas.
 
-    Returns:
-        anno_page (iiif-prezi3.skeleton.AnnotationPage): the AnnotationPage with
-            an Annotation and ResourceItem attached.
-    """
+        Args:
+            image_url (str): An HTTP URL which points to the image.
+            anno_id (str): An HTTP URL for the annotation to which the image will be
+                attached.
+            anno_page_id (str): An HTTP URL for the annotation page to which the
+                annotation will be attached.
 
-    def add_image(self, image_url, anno_id=None, anno_page_id=None):
+        Returns:
+            anno_page (iiif-prezi3.skeleton.AnnotationPage): the AnnotationPage with
+                an Annotation and ResourceItem attached.
+        """
         body = ResourceItem(id=image_url, type='Image')
         annotation = Annotation(id=anno_id, body=body, target=image_url, motivation='painting', type='Annotation')
```

Comment:
Just a comment, the type of the main IIIF object should be set by default from the JSON schema, so theoretically there should be no need for specifying it for Annotation and AnnotationPage, Canavses etc. etc.